spacy/default_config.cfg

```diff
@@ -26,6 +26,9 @@ batch_size = 1000
 [nlp.tokenizer]
 @tokenizers = "spacy.Tokenizer.v1"
 
+[nlp.vectors]
+@misc = "spacy.Vectors.v1"
```

Comment:
Should we have a separate registry for them?

Reply:
Yes, I guess we may as well just add `vectors`.